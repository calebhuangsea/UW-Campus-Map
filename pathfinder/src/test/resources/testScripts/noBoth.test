# test no both parent and child nodes

CreateGraph graph1

FindPath graph1 n1 n3