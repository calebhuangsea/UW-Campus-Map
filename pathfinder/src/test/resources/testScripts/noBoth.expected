# test no both parent and child nodes

created graph graph1

unknown: n1
unknown: n3